mesh/accountSettings/views.py

```diff
@@ -1,3 +1,30 @@
-from django.shortcuts import render
+from django.http import JsonResponse
 
-# Create your views here.
+from mesh.accountSettings.models import Settings
+
+
+def display_theme(request):
+    if request.method == "GET":
+        data = request.GET
+        response = {
+            "status": "success"
+        }
+        if data.get("accountID") is None:
+            response.update({"status": "error"})
+            response.update({"message": "Missing account ID."})
+            return JsonResponse(response)
+        else:
+            if Settings.objects.filter(accountID=int(data.get("accountID"))).exists():
+                settings = Settings.objects.filter(accountID=int(data.get("accountID")))
```

Comment:
This is somewhat expensive to call twice, no? I think this approach might make more sense rather than calling the same method twice:

settings = Settings.objects.filter(accountID=int(data.get("accountID")))

if settings: